### PR TITLE
feat:-Added a delete all filters action and added title to the add filter a…

### PR DESCRIPTION
**Pull request by Biki-das**
### changes

1. A small action added to delete all filters,  it can be useful to delete when someone has a bunch of them rather than clicking on them selectively.

2.There was no title on the `add filter` button so added the same

https://github.com/facebook/react/assets/72331432/af6c0725-05b6-4f46-b8c9-8079bd933a8e

review @hoxyq

### CODE REVIEW

File: packages/react-devtools-shared/src/devtools/views/Settings/ComponentsSettings.js [hunk 1]
```diff
@@ -434,11 +438,16 @@ export default function ComponentsSettings(_: {}): React.Node {
...
         </tbody>
       </table>
-
-      <Button onClick={addFilter}>
+      <Button onClick={addFilter} title="Add filter">
         <ButtonIcon className={styles.ButtonIcon} type="add" />
         Add filter
       </Button>
+      {componentFilters.length > 0 && (
+        <Button onClick={removeAllFilter} title="Delete All filters">
```
Comment (hoxyq) [inline packages/react-devtools-shared/src/devtools/views/Settings/ComponentsSettings.js:446]:
```suggestion
        <Button onClick={removeAllFilter} title="Delete all filters">
```

**Review by hoxyq — APPROVED** [review]
Great, thanks!

**Comment by Biki-das** [comment]
> Great, thanks!

thanks for looking into ti :-)

**Comment by hoxyq** [comment]
> @hoxyq is this good to go? or you have nits

Everything is good, just initially planned to merge it into next minor release, but now I think its okay to include it into next patch release.

**Pull request merged**